client/flutter/lib/pages/symptom_checker/question_pages/short_list_question_view.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_html/flutter_html.dart';
+import 'package:who_app/api/content/schema/symptom_checker_content.dart';
+import 'package:who_app/pages/symptom_checker/question_pages/previous_next_buttons.dart';
+import 'package:who_app/pages/symptom_checker/symptom_checker_model.dart';
+
+// A short list question supporting either single or multiple selection modes.
+class ShortListQuestionView extends StatefulWidget {
+  // Call back to set our answer
+  final SymptomCheckerPageDelegate pageDelegate;
+
+  // Model for our question
+  final SymptomCheckerPageModel pageModel;
+
+  const ShortListQuestionView({
+    Key key,
+    @required this.pageDelegate,
+    @required this.pageModel,
+  }) : super(key: key);
+
+  @override
+  _ShortListQuestionViewState createState() => _ShortListQuestionViewState();
+}
+
+class _ShortListQuestionViewState extends State<ShortListQuestionView> {
+  String _singleSelection;
+  Set<String> _multipleSelections = {};
+
+  bool get _allowsMultipleSelection {
+    return widget.pageModel.question.allowsMultipleSelection;
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    if (widget.pageModel.selectedAnswers.isNotEmpty) {
+      if (_allowsMultipleSelection) {
+        _multipleSelections = widget.pageModel.selectedAnswers;
+      } else {
+        _singleSelection = widget.pageModel.selectedAnswers.first;
+      }
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Padding(
+        padding: const EdgeInsets.all(32.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: <Widget>[
+            Spacer(),
+            Container(
+                width: 300,
+                child: Html(data: widget.pageModel.question.questionHtml)),
+            SizedBox(height: 24),
+            ...widget.pageModel.question.answers.map(_buildAnswerRow).toList(),
+            Spacer(flex: 3),
+            PreviousNextButtons(
+                showPrevious: widget.pageModel.questionIndex > 0,
+                enableNext: _isComplete,
+                onPrevious: _previous,
+                onNext: _next),
+            Spacer()
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildAnswerRow(SymptomCheckerAnswer answer) {
+    return GestureDetector(
+      onTap: () {
+        _selected(answer.id);
+      },
+      child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: <Widget>[
+            Material(
+              color: Colors.white,
+              child: _allowsMultipleSelection
+                  ? Checkbox(
+                      value: _multipleSelections.contains(answer.id),
+                      onChanged: (_) {},
+                    )
+                  : Radio<String>(
+                      groupValue:
+                          _singleSelection == answer.id ? answer.id : "",
```

Comment:
Again I think just `_singleSelection` should suffice

```suggestion
                      groupValue:  _singleSelection,
```